pkg/minikube/problem/err_map.go

```diff
@@ -55,7 +55,42 @@ var vmProblems = map[string]match{
 		Issues:        []int{6098},
 		ShowIssueLink: true,
 	},
-
+	"FILE_IN_USE": {
+		Regexp: re(`The process cannot access the file because it is being used by another process`),
+		Advice: "Another program is using a file required by minikube. If you are using Hyper-V, try stopping the minikube VM from within the Hyper-V manager",
+		URL:    "https://docs.docker.com/machine/drivers/hyper-v/",
+		GOOS:   []string{"windows"},
+		Issues: []int{7300},
+	},
+	"CREATE_TIMEOUT": {
+		Regexp: re(`create host timed out in 120`),
```

Comment:
is the 120 needed ? can we make the regex to search for a digit instead of 120 ?